Template2018/src/main/java/frc/team281/robot/subsystems/RealDriveSubsystem.java

```diff
@@ -79,28 +95,36 @@ public void initialize() {
 
 		this.rightTalonSettings = TalonSettingsBuilder.inverted(leftTalonSettings, false, true);
 
+		if (this.navX != null) {
+			while (this.navX.isCalibrating()) {
+				try {
+					Thread.sleep(50);  // 50ms
```

Comment:
perhaps extract 50ms to a constant?

Reply:
I don't think we are likely to use this constant anywhere else, so I'm inclined to leave it as is.

Reply:
i agree re-use is unlikely. however, Thread.sleep(NAVX_CALIBRATION_TIME_MS) is a more readable and cant become out of sync with the //50ms constant.

In the template i'm sensitive to setting good coding examples.  I'll merge now, we can make a cleanup pass through later.

Reply:
There you go showing me how much more readable that it.  If you think that's going to work, you're right.  Another change coming.

Reply:
;)